### PR TITLE
Make messages strings translateble.

### PR DESCRIPTION
**Pull request by frjo**

### CODE REVIEW

File: hypha/apply/activity/templates/messages/email/base.html [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {% block salutation %}Dear {{ user }},{% endblock %}
```
Comment (sandeepsajan0) [inline hypha/apply/activity/templates/messages/email/base.html]: I think we should add a `trans` tag for "Dear" as well?

Comment (frjo) [reply hypha/apply/activity/templates/messages/email/base.html]: Thanks, fix it now.

**Comment by sandeepsajan0** [comment]
Looks good to me.

**Comment by sandeepsajan0** [comment]
Just an idea/suggestion(it might not be related to this PR): 
1. What if we can pass the language code in urls?
It can help us to solve the "internal user's language issue" 
 >the ability for the “internal user” to choose what language Hypha is displayed in 

Because I think different users in the same org may want to use different languages.

2. I think it will be better to use Babel with `.pot` file to translate in other languages. Like it is [explained here](https://github.com/alexa/skill-sample-python-howto/blob/master/instructions/localization.md)?

**Comment by frjo** [comment]
Django has built in support with "makemessages" (extract all translation in to a po-file) and "compilemessages" (compile all translations in to binary mo-file) commands. Maybe Django uses babel to do this?

Planing to add Djangos "LocaleMiddleware" as well so Django can pick the language the browser wants.

**Pull request merged**